extensions/resteasy-reactive/rest/runtime/pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>junit-jupiter</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
We don't need this dependency.

Reply:
And there's the squash thingy...

Reply:
Oops, good point

Reply:
Dep removed. 

> And there's the squash thingy...

what  squash thing?

Reply:
sorry, squash the commits.

Reply:
No need to. One commit contains the author's original fix (with my amendment) and the other is just a minor code cleanup